src/Peachpie.CodeAnalysis/FlowAnalysis/ExpressionAnalysis.cs

```diff
@@ -164,12 +164,26 @@ public override T VisitStaticStatement(BoundStaticVariableStatement x)
                 // analyse initializer
                 Accept(v.InitialValue);
 
-                State.SetLessThanLongMax(local, (v.InitialValue.ConstantValue.HasValue && v.InitialValue.ConstantValue.Value is long && (long)v.InitialValue.ConstantValue.Value < long.MaxValue));
```

Comment:
just keep this line, try to not nest everything in several `if`s

Reply:
Yeah, this was a bit unfortunate, fixed in e8f7755671f70fa15bd3f069454a58a54ffb6d0a (just simplified a little).